Sources/StreamChatUI/Components.swift

```diff
@@ -223,7 +226,17 @@ public struct _Components<ExtraData: ExtraDataTypes> {
     /// The `UICollectionViewCell` subclass that shows channel information.
     public var channelCell: _ChatChannelListCollectionViewCell<ExtraData>.Type =
         _ChatChannelListCollectionViewCell<ExtraData>.self
-
+    
+    /// The `UICollectionViewCell` subclass that shows placeholder for channel information.
+    public var channelPlaceholderCell: _ChatChannelListPlaceholderCollectionViewCell<ExtraData>.Type =
+        _ChatChannelListPlaceholderCollectionViewCell<ExtraData>.self
+    
+    /// The view which is displayed when there is yet no channel for user in channelList.
+    public var channelListEmptyView: _ChatChannelListEmptyView<ExtraData>.Type = _ChatChannelListEmptyView<ExtraData>.self
```

Comment:
I think it is much better to use `UIView.Type` for both loading, error and empty state view parameters

```swift
public var channelListEmptyView: UIView.Type = _ChatChannelListEmptyView<ExtraData>.self
```